src/mysql/create.sql

```diff
@@ -34,6 +33,16 @@ CREATE TABLE Notification (
   REFERENCES User (id)
 );
 
+CREATE TABLE UserNotification (
```

Comment:
Can you also make sure to update the create.sql test file (I think it's under the test directory)? It would be good to make sure that those are aligned!

Reply:
Done!